api.md

```diff
@@ -266,71 +266,157 @@ Output:
 
 Status Code: 200 or 400
 
+### Create Project
+
+POST "/project_create"
+
+This will create a project with the current user as the admin.
+
+Only requires name and description of the project.
+
+Input: A JSON body with the following **required** parameters.
+
+```typescript
+{
+    name: string;
+    description: string;
+}
+```
+
+Status Code: 201 or 400
+
+### Get Project
+
+GET "/project_get"
+
+This will get the project's name, description and creation time.
+
+Input: A JSON body with the following **required** parameters.
+
+```typescript
+{
+    projectid: string;
+}
+```
+
+Status Code: 200 or 400
+
+### Create Task
```

Comment:
-> i think we need a task_get_all & event_get_all (or just data_get_all that does both), even though events not implemented yet just return empty or something
-> so that we can seed the data